erpnext/stock/doctype/stock_entry/stock_entry.py

```diff
@@ -270,8 +270,13 @@ def validate_qty(self):
 	def validate_fg_completed_qty(self):
 		if self.purpose == "Manufacture" and self.work_order:
 			production_item = frappe.get_value('Work Order', self.work_order, 'production_item')
+			allowance_percentage = flt(frappe.db.get_single_value("Manufacturing Settings",
+				"overproduction_percentage_for_work_order"))
```

Comment:
Where...are they using this variable then, if not here? :sweat_smile:

Reply:
So overproduction_percentage allows you to set the work order qty in the FG stock entry more than the Work Order.

So your work order says for 100. You can say 110 in the FG stock entry (In the parent; not the child table)